libraries/AP_Param/AP_Param.h

```diff
@@ -568,7 +568,9 @@ class AP_Param
 
     // return the persistent top level key for the ParamToken key
     static uint16_t get_persistent_key(uint16_t key) { return var_info(key).key; }
-    
+
+    bool allow_set_via_mavlink(uint16_t flags) const;
```

Comment:
can I be annoying and ask for a comment above this function?

Reply:
Done

Reply:
Hi @peterbarker, for some strange reason I don't see the comment still..

Reply:
My apologies, I managed to force-push over my own patch adding the comments:
![image](https://github.com/user-attachments/assets/55ae8040-2a68-4297-82d3-1dd1c233b0f9)

That first link shows the patches going in.